src/IonParserBinaryRaw.ts

```diff
@@ -131,6 +115,17 @@ const ivm_image_1 = IVM.binary[1];
 const ivm_image_2 = IVM.binary[2];
 const ivm_image_3 = IVM.binary[3];
 
+// stack pointer for _ts to store length and type information of container
+class stackPointer {
+  len: number;
```

Comment:
I know the original variable names were `len` and `type`, but since this is new code I'd like to spell `len` out in full.